model/command_webhook_test.go

```diff
@@ -46,7 +51,7 @@ func TestCommandWebhookIsValid(t *testing.T) {
 		err := h.IsValid()
 
 		if test.ExpectedError == "" {
-			assert.Error(t, err, "hook should be valid")
+			assert.Nil(t, err, "hook should be valid")
```

Comment:
This was false-passing before, as `Error`, but now that is checking for `Nil` is having the expected behavior.